services/github/github-package-lock-json.tester.js

```diff
@@ -0,0 +1,65 @@
+'use strict'
+
+const { ServiceTester } = require('../tester')
+const { semverRange } = require('../validators')
+
+const t = (module.exports = new ServiceTester({
+  id: 'GithubPackageLockJson',
+  title: 'GithubPackageLockJson',
+  pathPrefix: '/github/package-lock-json',
+}))
```

Comment:
Let's use the standard service helper
```suggestion
const t = (module.exports = require('../tester').createServiceTester())
```